examples/complete/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "1.9.0"
+  required_version = "1.11.4"
```

Comment:
@gberenice hm... I'm glad this isn't updating the root level `versions.tf`... I wonder what we should do with these sort of automated updates going forward. We don't really need to care about the examples/ tf versions. Maybe we have it ignore that folder?